scripts/RT/wepp_worker.py

```
@@ -42,7 +43,13 @@ def run_wepp(payload: WeppJobPayload):
     # We run timeout to keep things from hanging indefinitely, we tried 60
     # seconds but it was too short as sometimes latency happens.
     with subprocess.Popen(
-        ["timeout", "-s", "9", "600", sanitize_exe(payload.weppexe)],
+        [
+            "timeout",
+            "-s",
+            "9",
+            "600",
+            str(sanitize_exe(BINPATH / payload.weppexe)),
+        ],
```

Ruff (0.15.13)

[error] Starting a process with a partial executable path

(S607)